### DISCUSSION TITLE
Welcome to CellMAPtracer Discussions!

### DISCUSSION THREAD

**Discussion by wleoncio** [ocbe-uio/CellMAPtracer#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shreyasimodak** [comment]
how do I load the FUCCI plug-in on the CellMAPtracer MATLAB app?

**Reply by wleoncio** [reply]
Not so sure, right now I click the "load FUCCI plugin" button and nothing seems to happen. Is this what you also experience? What OS are you running CellMAPtracer on?

**Reply by shreyasimodak** [reply]
I don't see any "load FUCCI plugin" option anywhere in the CellMAPtracer main window. I am running on Mac and the MATLAB app.

**Reply by wleoncio** [reply]
Did you manage to open the graphical interface? Could you please share a screenshot of it, and let me know what version of CellMAPtracer you're running? This is what I see on Linux:

![bilde](https://user-images.githubusercontent.com/8234768/227862559-f4869728-ae12-4dbb-81a7-75720dc4b911.png)

**Reply by shreyasimodak** [reply]
The GUI is opening but it does not show the FUCCI option. It looks like this:
<img width="1024" alt="Screenshot 2023-03-27 at 5 32 13 PM" src="https://user-images.githubusercontent.com/128902390/227936396-2b0242ab-6e19-4412-9dc3-0a4802a534ab.png">
The version is 1.00, the one that is available in the "app" section of the main folder

**Reply by wleoncio** [reply]
OK, looks like there are a couple of things to address here. I'll open issues related to them, hopefully the developers can fix and release a new version soon.

**Reply by wleoncio** [reply]
Issues #10 and #11 opened, looking forward to hearing from the devs!
